tests/ga/test_extension.py

```diff
@@ -79,103 +81,129 @@ def raise_ioerror(*args):
     e.errno = EIO
     raise e
 
-
 class TestExtensionCleanup(AgentTestCase):
+
```

Comment:
Reformatted the `TestExtensionCleanup` class to test the public interfaces rather than testing the internal private functions.